repositoryPassword.txt

```diff
@@ -0,0 +1 @@
+tiagodinis123
```

Comment:
oh god is my repository's password frick

Reply:
You might be interested in knowing that there is a file called `.git/info/exclude`. It works exactly like `.gitignore`, but it's local to a single clone of a repository. It never gets committed and pushed.
Adding the line `repositoryPassword.txt` to your `.git/info/exclude` is an easy way to prevent you accidentally committing it without having to modify the repo's `.gitignore`.

Reply:
hmm thx